etiqet-core/src/main/java/com/neueda/etiqet/fixture/EtiqetFixtures.java

```diff
@@ -562,6 +562,12 @@ public void waitForAMessageAsTimesOut(String messageType, String messageName) th
 		handlers.validateMessageTypeDoesNotExistInResponseMap(messageName);
 	}
 
+	@And("^wait for \"([^\"]*)\" to receive a \"([^\"]*)\" message times out$")
```

Comment:
The grammar reads a wee bit weird here, also how long is a timeout? Is it configurable, or fixed?